Technical Review - Survey status vocabulary

KellyVance opened the pull request.

KellyVance commented:
See dataset themes discussion on use of the term _Mining_

johnmckellar commented:
Yes, I had an issue with 'mining' in the terminology. I discussed this with David, and we could not quickly find a suitable alternative, taking the term to refer to the mining industry (and the Mines Dept) in general. I overcame the issue with the definition: The stage in the lifecycle of an exploration survey or activity (including research activities).

But, a suitable alternative would be preferable. Until I understood what the term is supposed to cover, I was misled by it, hence the definition.

dxwell commented:
> See dataset themes discussion on use of the term _Mining_

From https://en.wikipedia.org/wiki/Mining

Mining is the extraction of valuable minerals or other geological materials from the Earth, usually from an ore body, lode, vein, seam, reef or placer deposit. These deposits form a mineralized package that is of economic interest to the miner.

Ores recovered by mining include metals, coal, oil shale, gemstones, limestone, chalk, dimension stone, rock salt, potash, gravel, and clay. Mining is required to obtain any material that cannot be grown through agricultural processes, or feasibly created artificially in a laboratory or factory. Mining in a wider sense includes extraction of any non-renewable resource such as petroleum, natural gas, or even water.

dxwell commented on vocabularies/mining-survey-status.ttl:
```diff
@@ -0,0 +1,63 @@
+
+sstatus:abandoned a skos:Concept;
+  skos:notation "ABAND"@en;
+  skos:definition "A survey or activity has been abandoned, because, although commenced, it was not completed due to unforeseen issues or for other reasons."@en;
```
Should "although commenced" be removed? e.g. a company lodged a Notice of Intention to Drill and then never drilled.

KellyVance replied:
changed

dxwell commented on vocabularies/mining-survey-status.ttl:
```diff
@@ -0,0 +1,63 @@
+  dct:modified "2019-04-09T03:34:27.949Z"^^xsd:dateTime;
+  skos:definition "The stage in the lifecycle of an exploration survey or activity (including research activities)"@en;
+  skos:prefLabel "Mining Survey or Activity Status"@en .
+
```
Add:  
dct:creator "Geological Survey of Queensland" ;

KellyVance replied:
done

dxwell commented on vocabularies/mining-survey-status.ttl:
```diff
@@ -0,0 +1,63 @@
+  dct:created "2019-04-09T03:31:01.023Z"^^xsd:dateTime;
+  dct:modified "2019-04-09T03:34:27.949Z"^^xsd:dateTime;
+  skos:definition "The stage in the lifecycle of an exploration survey or activity (including research activities)"@en;
+  skos:prefLabel "Mining Survey or Activity Status"@en .
```
What are some examples of "Mining Activity"? For me, "Activity" is too broad a term.  
Survey: a general view, examination, or description of someone or something.  
Activity: a thing that a person or group does or has done

johnmckellar replied:
To me, 'Mining Survey' is too narrow, and, more importantly, too misleading. In my mind, it conjures up someone doing some sort of survey at a mine—surely, this is not what's intended?

Yes, activity is broad, and this is why I prefer the term, as the vocab can then encompass any thing geological, whether it's to do with mining, petroleum, CSG or some researcher working on the mineralisation of cyanophytes in the Proterozoic of NW Queensland, or, indeed, just studying these bacteria from morphological, nomenclatural, environmental and/or evolutionary points of view (i.e. an academic study).

Otherwise, we will end up with a plethora of overlapping and thus somewhat useless vocabs!!

After I ascertained the function of the vocab, I proposed the definition: "The stage in the lifecycle of an exploration survey or activity (including research activities)". I gave this definition to overcome the limitations of the terms 'Mining' and 'Survey', and, David, it was based on your original definition.

Essentially, I see this vocab as an activity tracker, whether it's to do with mining, mapping, an exploration project  or otherwise, but to embrace all things/work geological. Thus, I maintain we should also move away from the misleading term 'Mining'. Perhaps: 'Geological activity status' and focus on the attendant definition?

DavidCrosswellGSQ commented:
**Concepts:**
Intended
In-progress
Survey completed
Interim report submitted
Final report submitted
On-hold
Abandoned

DavidCrosswellGSQ commented:
Change title to "Survey Status"
Change URI to /def/survey-status

KellyVance commented:
As per conversation
Adjusted concepts to reflect list above. i.e.
   - Deleted outstanding data category
   - Added Final Report received category
Changed title to Survey status
changed URI to /def/survey-status
@DavidCrosswellGSQ @dxwell @johnmckellar

KellyVance commented:
> Hi Vance,
> 
> The stage in the lifecycle of a conducting a survey.
> 
> Omit 'a' before 'conducting'.

Changed.

DavidCrosswellGSQ commented on vocabularies/mining-survey-status.ttl:
```diff
@@ -0,0 +1,65 @@
+  dct:created "2019-04-09T03:31:01.023Z"^^xsd:dateTime;
+  dct:modified "2019-04-09T03:34:27.949Z"^^xsd:dateTime;
+  dct:creator "Geological Survey of Queensland" ;
+  skos:definition "The stage in the lifecycle of conducting a survey"@en;
```
Add a full stop at end of sentence.

johnmckellar replied:
done

DavidCrosswellGSQ commented on vocabularies/mining-survey-status.ttl:
```diff
@@ -0,0 +1,65 @@
+﻿@prefix dct: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> ..
```
Delete second full stop

johnmckellar replied:
done

dxwell commented:
Approved.

KellyVance commented:
@dxwell can you please approve via the _review changes_ function?

johnmckellar commented:
Thanks Vance
